### PR TITLE
#571/Add time between slow connection warnings

### LINKED ISSUE #571: Add time between slow connection warnings

### PR DESCRIPTION
**Pull request by gbudau**
close #571

### CODE REVIEW

File: webapp/src/shared/components/Game/hooks/useOnlineGame.tsx [hunk 1]
```diff
@@ -29,6 +29,7 @@ const SHOULD_CONFIGURE_GAME = 'shouldConfigureGame';
 const GET_SHOULD_CONFIGURE_GAME = 'getShouldConfigureGame';
 const GAME_CONFIG_SUBMIT = 'gameConfigSubmit';
 const MAX_CONNECTION_LAG_IN_MS = 100;
+const WARN_INTERVAL_IN_MS = 5000;
```
Comment (vdedios) [inline webapp/src/shared/components/Game/hooks/useOnlineGame.tsx:32]: I'd configure this as 10 seconds, 5 is still too low

**Review by vdedios — APPROVED** [review]
Thanks!

File: webapp/src/shared/components/Game/hooks/useOnlineGame.tsx (hunk 1 above)
Comment (gbudau) [reply webapp/src/shared/components/Game/hooks/useOnlineGame.tsx]: const usersColumns = Object.values(userKeys)
      .map(
        (column) => `users.${column} as user${capitalizeFirstLetter(column)}`,
      )
      .join(', ');

**Comment by gbudau** [comment]
Thanks!

**Pull request merged**